web/src/engine/osk/src/input/gestures/browser/subkeyPopup.ts

```diff
@@ -6,7 +6,7 @@ import VisualKeyboard from '../../../visualKeyboard.js';
 import { DeviceSpec, ActiveSubKey, KeyDistribution, ActiveKeyBase } from '@keymanapp/keyboard-processor';
 import { ConfigChangeClosure, GestureRecognizerConfiguration, GestureSequence, PaddedZoneSource, RecognitionZoneSource } from '@keymanapp/gesture-recognizer';
 import { GestureHandler } from '../gestureHandler.js';
-import { CorrectionLayout, CorrectionLayoutEntry, distributionFromDistanceMaps, keyTouchDistances } from '@keymanapp/input-processor';
+import { CorrectionLayout, CorrectionLayoutEntry, distributionFromDistanceMaps, keyTouchDistances } from 'keyman/engine/osk';
```

Comment:
I feel it's a bit odd to have a class use an external-style import pattern for something within the same package, but maybe that's just me.  (This is hardly the only place with this sort of change.)